src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import propTypes from 'prop-types';
+
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+import { getUser } from '../../api/api';
+
+export const CurrentUser = ({ userId, clearUser }) => {
+  const [user, setUser] = useState();
+
+  useEffect(() => {
+    getUser(userId)
+      .then(setUser);
+  });
```

Comment:
```suggestion
  useEffect(() => {
    getUser(userId)
      .then(setUser);
  }, []);
```
If you want to use it like `componentDidMount` use an empty dependencies array as a second argument in useEffect